ionic-jhipster-example/src/pages/login/login.ts

```diff
@@ -47,4 +52,47 @@ export class LoginPage {
       toast.present();
     });
   }
+
+  doLoginFacebook() {
+    
+    this.user.loginFacebook()
+      .then((res: FacebookLoginResponse) => {
+       
+
+        this.user.accessFacebookApi()
+          .then((res) => {
+           
+
+            this.account = {
+              username: res.email,
+              password: this.user._userFBId
+            };
+
+            this.doLogin();
+
+          })
+          .catch((err) => {
+            
+
+            this.presentToastMessage('Error login to Facebook');
```

Comment:
Is there an error that comes back from Facebook? If so, we should `console.error()` it at the very least.